src/mapping/config/MappingConfiguration.cpp

```diff
@@ -195,10 +205,18 @@ MappingConfiguration::MappingConfiguration(
                                .setDocumentation("Sets kind of preallocation for PETSc RBF implementation")
                                .setOptions({PREALLOCATION_ESTIMATE, PREALLOCATION_COMPUTE, PREALLOCATION_OFF, PREALLOCATION_SAVE, PREALLOCATION_TREE});
 
+  auto verticesPerCluster = XMLAttribute<int>(ATTR_VERTICES_PER_CLUSTER)
+                                .setDocumentation("Vertices per cluster (partition) applied in the rbf partition of unity method.");
+  auto relativeOverlap = makeXMLAttribute(ATTR_RELATIVE_OVERLAP, 0.3)
+                             .setDocumentation("Value between 0 and 1 indicating the relative overlap between clusters. A value of 0.3 is usually a good trade-off between accuracy and efficiency.");
+  auto projectToInput = XMLAttribute<bool>(ATTR_PROJECT_TO_INPUT)
+                            .setDocumentation("If enabled, places the cluster centers at the closest vertex of the input mesh. Should be enabled in case of non-uniform point distributions such as for shell structures.");
```

Comment:
Hm, I guess in most of the cases it won't hurt. For the Halton points, e.g. it might hurt in combination with a low number of vertices per cluster. In many cases it is surprisingly beneficial..

Reply:
See comment below